client/reader/following/style.scss

```diff
@@ -5,6 +5,76 @@
 .following__search.card.is-compact {
 	padding: 0;
 	margin-bottom: 16px;
-	box-shadow: 0 0 0 2px transparentize( lighten( $gray, 20% ), .5 ), 0 1px 2px lighten( $gray, 30% );
+	box-shadow: 0 0 0 2px transparentize( lighten( $gray, 20% ), 0.5 ), 0 1px 2px lighten( $gray, 30% );
 	z-index: z-index( 'root', '.reader-following-search' );
 }
+
+.following__intro {
+	background: url( '/calypso/images/reader/reader-intro-background.jpg' );
+	background-position: 100% 20%;
+	min-height: 280px;
+	background-repeat: no-repeat;
+	background-size: cover;
+
+	@media ( max-width: 900px ) {
+		background-position: 75% 0;
+	}
+
+	@media ( max-width: 500px ) {
+		background-position: 65% 0;
+		min-height: 260px;
+	}
+}
+
+.following__intro-header {
+	display: flex;
+	width: 100%;
+}
+
+.following__intro-close {
+	margin-left: auto;
+	color: #fff;
+	margin-top: 16px;
+	margin-right: 16px;
+	background-color: rbga( 0, 0, 0, 1 );
```

Comment:
`background` is more performant I believe. 

What does this background color do? Is it a background color on the icon? Just trying to determine if this should use a variable.